samples/cpp/tutorial_code/ImgTrans/HoughLines_Demo.cpp

```diff
@@ -15,22 +15,27 @@ using namespace std;
 /// Global variables
 
 /** General variables */
-Mat src, edges;
+Mat src, canny_edge, sobel_edge;
 Mat src_gray;
-Mat standard_hough, probabilistic_hough;
+Mat standard_hough, probabilistic_hough, weighted_hough;
 int min_threshold = 50;
 int max_trackbar = 150;
+int weightedhough_max_trackbar = 1e+5;
```

Comment:
MS Visual Studio reports warning here. Please use 100000. See CI logs for details.

Reply:
Oh I'm very sorry. Now I'm fixing and checking the build procedure. I will push after I confirmed it build without error.

Reply:
I think I fixed the 2 errors in the CI.
But github workflow seems only run on branch 4.x, so I couldn't confirmed the fixation. 
Could you check it by running CI?